.github/workflows/test.yml

```diff
@@ -20,10 +20,6 @@ jobs:
       - uses: ApeWorX/github-action@v1
         with:
           python-version: '3.9'
-          ape-version-pin: '==0.5.2'
-
-      - name: Install Vyper
```

Comment:
I am now realizing this is likely down because otherwise vvm downloads vyper from Github